packages/cicero-ui/src/lib/components/Clause/index.js

```diff
@@ -69,6 +74,42 @@ const ClauseComponent = React.forwardRef((props, ref) => {
     clauseIconColor: clauseProps.ICON_HOVER_COLOR,
   };
 
+  const dragIconProps = {
+    'aria-label': dragIcon.type,
+    width: '12px',
+    height: '22px',
+    viewBox: '0 0 12 22',
+  };
+
+  const upIconProps = {
+    'aria-label': upIcon.type,
+    width: '16px',
+    height: '12px',
+    viewBox: '0 0 16 12',
+  };
+
+  const downIconProps = {
+    'aria-label': downIcon.type,
+    width: '16px',
+    height: '12px',
+    viewBox: '0 0 16 12',
+  };
+
+  const handleClick = (event, down) => {
+    const node = ReactEditor.toSlateNode(props.editor, event.target);
+    const path = ReactEditor.findPath(props.editor, node);
+    const range = Editor.range(props.editor, path);
+    const documentEnd = Editor.end(props.editor, []);
+    const newPath = down
+      ? [Math.max(range.anchor.path[0], range.focus.path[0]) + 1]
+      : [Math.max(range.anchor.path[0], range.focus.path[0]) - 1];
```

Comment:
This might not be an improvement, but I had a thought:

```js
const newPath = [Math.max(range.anchor.path[0], range.focus.path[0]) + (down ? 1 : -1)]
```